src/codeOfProgram/main/DeckOfCards.java

```diff
@@ -0,0 +1,17 @@
+package codeOfProgram.main;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeckOfCards {
```

Comment:
This class should be so called entity class, with field that represents Deck and methods to work with Deck.

Reply:
+